src/pages/edit-profile/EditProfile.styles.ts

```diff
@@ -1,6 +1,9 @@
 import { TypographyVariantEnum } from '~/types'
 
 export const styles = {
+  sectionsWrapper: {
+    px: { lg: '156px' }
```

Comment:
Same here

Reply:
I've also delete this sectionsWrapper style with puddings
I think it will be better to fix width for all page containers (<PageWrapper>)
Because now each page have different container-width not the same as on mockup